community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -380,9 +380,15 @@ def configure_dirs():
             sl.unlink()
         sl.symlink_to(tgt)
 
-    for f in ("sort_nodes.py",): # copy auxiliary scripts
-        dst = Path(lookup().cfg.slurm_bin_dir) / f
-        shutil.copyfile(util.scripts_dir / f, dst)
+    # copy auxiliary scripts
+    for dst_folder, src_file in ((lookup().cfg.slurm_bin_dir,
+                                  Path("sort_nodes.py")),
+                                 (Path(dirs.custom_scripts / "task_prolog.d"),
```

Comment:
nit. `dirs.custom_scripts` is already path, 
Replace `Path(dirs.custom_scripts / "task_prolog.d")` -> `dirs.custom_scripts / "task_prolog.d"`

Reply:
Fixed.